src/EasyRdf/EasyRdfClient.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\EasyRdf;
+
+use App\Rdf\Client;
+use App\Rdf\Exception\InvalidSparqlQuery;
+use App\Rdf\SparqlQuery;
+use App\Rdf\Triple;
+use EasyRdf_Graph;
+use EasyRdf_Sparql_Client;
+
+class EasyRdfClient implements Client
```

Comment:
Do we need to use EasyRdf in the class name if we wrap it anyway? perhaps RdfClient would be more appropriate?

Reply:
I just added it to be explicit. that this is an Implementation of Client using EasyRdf. Though it is already in Namespace. So I can change it to RdfClient. I don't think that we are going to add dozen of different clients.